src/tests/ftest/ior/small.yaml

```diff
@@ -13,13 +13,19 @@ server_config:
       fabric_iface: ib0
       fabric_iface_port: 31317
       log_file: daos_server0.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,io,md,epc
       storage: auto
     1:
       pinned_numa_node: 1
       nr_xs_helpers: 1
       fabric_iface: ib1
       fabric_iface_port: 31417
       log_file: daos_server1.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
```

Comment:
im not sure if this PR is pushed for landing, but we should not increase log levels on tests like that. logs are huge as is and increasing logs on IOR would make things explode.

Reply:
This is not for landing, just for collect more debug logs, not care about the performance.